blocks/newslist/newslist.css

```diff
@@ -36,21 +36,45 @@
   margin-right: auto;
 }
 
+@media (min-width: 1000px) {
+  .newslist .newslist-container .newslist-item h4 a {
+    font-size: 20px;
+  }
+}
+
 .newslist .newslist-container .newslist-item .newslist-item-description {
   font-family: var(--header-font-family);
   font-size: var(--body-font-size-xxs);
   color: #000;
-  margin-bottom: 20px;
   padding: 0 0 20px;
-  line-height: 1.83;
+  line-height: 1.6;
+}
+
+.newslist .newslist-container .newslist-item .newslist-item-description p {
+  margin-bottom: 0;
+}
+
+@media (min-width: 767px) {
```

Comment:
we'd like to avoid to have this media query, default should be mobile. Override accordingly in desktop break point.

Reply:
hi @ravkiran, my changes are for tablet view and desktop view. and also, I change the viewport value from 767 to 768.